project/settings.py

```diff
@@ -97,6 +97,7 @@
     'providers.ca.umontreal',
     'providers.ca.uwo',
     'providers.ch.cern',
+    'providers.cy.ac.cut',
```

Comment:
Would it still be accurate to remove the `ac` directory so that there isn't an extra folder involved?